crates/jstzd/src/task/jstzd.rs

```diff
@@ -148,6 +178,18 @@ impl Jstzd {
         }
         Ok(())
     }
+
+    /// Wait for the baker to bake at least `level` blocks.
+    async fn wait_for_block_level(node_endpoint: &Endpoint, level: i64) -> Result<()> {
+        let ready = retry(10, 1000, || async {
+            Ok(get_block_level(&node_endpoint.to_string()).await? >= level)
```

Comment:
Nit: I think you could just map to `bool` like `.await.map(|l| l >= level)` without wrapping in `Ok()`

Reply:
neat!